### PR TITLE
sdk: Set chainId for tx payload when sending

### PR DESCRIPTION
**Pull request by miguelmota**
- [x] set `chainId` when sending transaction based on method passed in chain parameter (ie `send` on HopBridge) or chain property on class instance (ie `Token`, `AMM`)

### CODE REVIEW

**Comment by miguelmota** [comment]
Ready for review

File: packages/sdk/src/Base.ts [hunk 1]
```diff
@@ -164,6 +164,11 @@ class Base {
     }
   }
 
+  sendTransaction (transactionRequest: any, chain: TChain) {
```
Comment (kangarang) [inline packages/sdk/src/Base.ts]: minor suggestion: set type to ethers `TransactionRequest`

```suggestion
  sendTransaction (transactionRequest: TransactionRequest, chain: TChain) {
```

Comment (kangarang) [reply packages/sdk/src/Base.ts]: or `transactionRequest: TransactionRequest | ContractTransaction`

**Review by kangarang — APPROVED** [review]
other than the 1 comment, LGTM

File: packages/sdk/src/Base.ts (hunk 1 above)
Comment (miguelmota) [reply packages/sdk/src/Base.ts]: thanks :+1: 

Resolved by https://github.com/hop-protocol/hop/pull/412/commits/1b993a50ec48a897213310a8e03f7a0f08c6c058

**Pull request merged**